docs/faq/README.md

```diff
@@ -7,7 +7,6 @@ In this section you can find answers to frequently asked questions regarding jav
 
 **Deep dives:**
 
-- [Component Libraries](./component-libraries.html)
 - [Rerender not triggered](./rerender.html)
```

Comment:
you wanna link to the events faq here right?

Reply:
Interesting...how does something get into the `[[toc]]` annotation on these pages?

Reply:
it's here
https://github.com/open-wc/open-wc/blob/master/docs/.vuepress/config.js#L128

Reply:
Ok, this this iteration is good to go, now!